### PR TITLE
Tailwind css

### PR DESCRIPTION
**Pull request by 47codemonkey**
Added tailwind css packeges and configuration. Deleted .css files. Css variables were left without changes. Redundant css styles were deleted

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "@types/redux-mock-store": "^1.0.6",
     "@typescript-eslint/eslint-plugin": "^5.59.0",
     "@typescript-eslint/parser": "^5.59.0",
+    "autoprefixer": "^10.4.20",
```
Comment (dmytro-honch) [inline package.json:62]: c'mon 2024. Are we really still need it?

File: package.json [hunk 2]
```diff
@@ -72,12 +73,14 @@
     "eslint-plugin-storybook": "^0.8.0",
     "gh-pages": "^6.1.1",
     "husky": "^8.0.0",
+    "postcss": "^8.4.45",
```
Comment (dmytro-honch) [inline package.json:76]: ditto

File: postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export default {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
```
Comment (dmytro-honch) [inline postcss.config.js:4]: I guess you just copied this config from an interesting article and that's why we're having `autoprefixer` and `postcss`.

File: src/app.css [hunk 4]
```diff
@@ -23,17 +20,14 @@
     --secondary-element-color-lt: rgba(0, 0, 0, 0.1);
     --accent-text-color-lt: #0080c8;
```
Comment (dmytro-honch) [inline src/app.css:21]: as far as I know the Tailwind has themes switching configuration under the hood. The same rule for modals and so. Don't consider it as "just a feature what gives me ability to write css in jsx" it's wrong. And I really think it digression, but who asks me :)

File: src/components/_common/iconicButton/iconicButton.tsx [hunk 5]
```diff
@@ -11,7 +9,7 @@ export const IconicButton = ({ onClickHandler, children }: IconicButtonProps) =>
   <button
     type="button"
     data-testid="iconic-button"
-    className="header-buttons switch-theme-button"
+    className="hover:text-[var(--primary-text-color)] absolute md:top-10 md:right-10 top-5 right-5 cursor-pointer z-[1001] p-[6px] rounded-full leading-[1] text-icon-font-size bg-[var(--primary-element-color)]"
```
Comment (dmytro-honch) [inline src/components/_common/iconicButton/iconicButton.tsx:12]: strong weed, bro. See comment about themes and don not ever do it.

File: src/components/about/about.tsx [hunk 6]
```diff
@@ -1,41 +1,46 @@
...
+      About
+    </h2>
+    <div className="pr-5 pl-5 md:pr-10 md:pl-10">
+      <p className="my-4">
         About Welcome to our non-profit project, "Exploring the Solar System"! We are Dmytro Honcharov and Oleksii
         Moroz, two passionate software developers with a shared fascination for the vastness and beauty of our cosmic
         neighborhood.
       </p>
-      <p>
+      <p className="my-4">
```
Comment (dmytro-honch) [inline src/components/about/about.tsx:12]: what a hack?! Don't you heard my class regarding styles for paragraphs??? Delete it!

File: src/components/about/about.tsx [hunk 7]
```diff
@@ -1,41 +1,46 @@
...
       </p>
-      <p>We used here next resources:</p>
-      <a href="https://en.wikipedia.org/wiki/List_of_Solar_System_objects_by_size" target="_blank" rel="noreferrer">
+      <p className="mt-4">We used here next resources:</p>
+      <a
+        className="text-[var(--accent-text-color)]"
+        href="https://en.wikipedia.org/wiki/List_of_Solar_System_objects_by_size"
+        target="_blank"
+        rel="noreferrer"
+      >
```
Comment (dmytro-honch) [inline src/components/about/about.tsx:43]: ╭(ʘ̆~◞౪◟~ʘ̆)╮

File: tailwind.config.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
```
Comment (dmytro-honch) [inline tailwind.config.js:4]: what for we have this word here?
¯\(°_o)/¯

**Review by dmytro-honch — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (47codemonkey) [reply package.json:62]: resolved

File: package.json (hunk 2 above)
Comment (47codemonkey) [reply package.json:76]: resolved

File: postcss.config.js (hunk 3 above)
Comment (47codemonkey) [reply postcss.config.js:4]: deleted

File: package.json (hunk 1 above)
Comment (47codemonkey) [reply package.json:62]: resolved

File: package.json (hunk 2 above)
Comment (47codemonkey) [reply package.json:76]: resolved

File: postcss.config.js (hunk 3 above)
Comment (47codemonkey) [reply postcss.config.js:4]: deleted

File: src/components/about/about.tsx (hunk 6 above)
Comment (47codemonkey) [reply src/components/about/about.tsx:12]: resolved

File: src/components/about/about.tsx (hunk 7 above)
Comment (47codemonkey) [reply src/components/about/about.tsx:43]: (⊙_☉)

File: tailwind.config.js (hunk 8 above)
Comment (47codemonkey) [reply tailwind.config.js:4]: which one?

**Pull request closed without merging**